lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -8,6 +8,18 @@ def define
 
       config = Calculators::BenefitCapCalculatorConfiguration.new
 
+      # Routing question
+      multiple_choice :choose_cap_to_calculate? do
+        option :default
+        option :future
+
+        save_input_as :chosen_cap
```

Comment:
The name of the method reflects what the question is doing, i.e. that it's asking the user to select a flow even though the user is unaware that is what is happening.